### ISSUE TITLE
预训练模型？

### ISSUE THREAD

**Issue by juntaosun** [myshell-ai/MeloTTS#66]
嗨，感谢你们出色的工作，我想训练自己的数据集，但没有找到该项目的预训练模型（底模），它还在上传吗？

**Comment by jeremy110** [comment]
download_utils.py當中有提供底模，當你使用bash train.sh會自動下載

**Comment by juntaosun** [comment]
感谢指导，已经开始在尝试训练了，预训练底模含几种语言呢？

**Comment by Okkband** [comment]
我也是， 训练完之后，一只说胡话，音色是对的，但是文字都不对。

**Comment by jinqiupeter** [comment]
> 我也是， 训练完之后，一只说胡话，音色是对的，但是文字都不对。

我用一个八分钟的音频训练出来也是一直说胡话，现在在尝试用两个小时的素材训练。

你们用多少素材训练的？作者在另一个issue里面说至少要几个小时的音频

**Comment by jeremy110** [comment]
@juntaosun @Okkband 
照範例是說支援六種語言，我另外一位同事有訓練中文，大概是10小時

**Comment by Okkband** [comment]
但是GPT-Sovits只需要几十句话就能得到效果好的音色，只不过GPT-Sovits推理会慢很多。

**Comment by juntaosun** [comment]
经过验证，预训练底模为：＂英文底模＂。

---

虽然 symbols 支持其它语言，但（底模的能力非常有限），（除了英文）它并没在其它语言上进行预训练。
所以，它是无法快速微调出其它语言的，比如中文，因此推理发音基本不对。

**Comment by Coder0015** [comment]
> @juntaosun @Okkband @jinqiupeter 照範例是說支援六種語言 我另外一位同事有訓練中文，大概是10小時，可以出來正常的文字及音色，但韻律差了一點

想请教一下，是因为数据集的量或是因为config文件参数的问题吗？我不管怎么训，吐字都不怎么标准且有些字连发音都完全不对。

**Comment by jeremy110** [comment]
@Coder0015 或許跟這個有關係https://github.com/myshell-ai/MeloTTS/issues/83

**Comment by insufficient-will** [comment]
有更新吗？从gpt-sovits跳过来的。我训练了5小时，中文训练集（和给gpt-sovits的一样），结果是说胡话。确信是加长时间（十小时？）可以解决吗？谢谢！

**Comment by steven8274** [comment]
I also met the problem of speaking nonsense.I finally found the problem is caused by  big batchsize.When I change the batchsize from 20 to 6(default value)，the result of training is much better.After about 700 epoch, most chinese charactors the model can speak correctly.My training dataset has 1500 audio samples which contain about 15000 seconds audio.

**Comment by jeremy110** [comment]
I'm not quite sure about the batch size, as I'm using the default value. Adjusting the batch size might require tuning the learning rate or some other parameters. In my case, after generating three or four checkpoints, the output sounds correct, but the pronunciation and rhythm are slightly off. By training for nearly 300 epochs, those issues are almost resolved.

**Comment by steven8274** [comment]
> I'm not quite sure about the batch size, as I'm using the default value. Adjusting the batch size might require tuning the learning rate or some other parameters. In my case, after generating three or four checkpoints, the output sounds correct, but the pronunciation and rhythm are slightly off. By training for nearly 300 epochs, those issues are almost resolved.

Thanks for sharing your traing experience！I have another question about multi-language training, could you help me?Thanks a lot in advance!https://github.com/myshell-ai/MeloTTS/issues/201#issue-2584637792

**Comment by KeZhang19** [comment]
> I also met the problem of speaking nonsense.I finally found the problem is caused by big batchsize.When I change the batchsize from 20 to 6(default value)，the result of training is much better.After about 700 epoch, most chinese charactors the model can speak correctly.My training dataset has 1500 audio samples which contain about 15000 seconds audio.

Excuse me, is your training process based on what is written on this open-source project?

**Comment by steven8274** [comment]
> > I also met the problem of speaking nonsense.I finally found the problem is caused by big batchsize.When I change the batchsize from 20 to 6(default value)，the result of training is much better.After about 700 epoch, most chinese charactors the model can speak correctly.My training dataset has 1500 audio samples which contain about 15000 seconds audio.
> 
> Excuse me, is your training process based on what is written on this open-source project?

Yes, completely follow [official training steps](https://github.com/myshell-ai/MeloTTS/blob/main/docs/training.md)

**Comment by KeZhang19** [comment]
> > > I also met the problem of speaking nonsense.I finally found the problem is caused by big batchsize.When I change the batchsize from 20 to 6(default value)，the result of training is much better.After about 700 epoch, most chinese charactors the model can speak correctly.My training dataset has 1500 audio samples which contain about 15000 seconds audio.
> > 
> > 
> > Excuse me, is your training process based on what is written on this open-source project?
> 
> Yes, completely follow [official training steps](https://github.com/myshell-ai/MeloTTS/blob/main/docs/training.md)

Thank you, how many speakers were used in your dataset?I also want to train a Chinese model now, so I would like to learn more about it.

**Comment by KeZhang19** [comment]
> > > I also met the problem of speaking nonsense.I finally found the problem is caused by big batchsize.When I change the batchsize from 20 to 6(default value)，the result of training is much better.After about 700 epoch, most chinese charactors the model can speak correctly.My training dataset has 1500 audio samples which contain about 15000 seconds audio.我还遇到了胡说八道的问题。我最后发现问题是 batchsize 大引起的。当我将 batchsize 从 20 更改为 6（默认值）时，训练结果要好得多。经过大约 700 个 epoch 后，大多数中国字符模型都能正确说话。我的训练数据集有 1500 个音频样本，其中包含大约 15000 秒的音频。
> > 
> > 
> > Excuse me, is your training process based on what is written on this open-source project?请问，您的培训过程是基于这个开源项目上所写的内容吗？
> 
> Yes, completely follow [official training steps](https://github.com/myshell-ai/MeloTTS/blob/main/docs/training.md)是的，完全遵循[官方培训步骤](https://github.com/myshell-ai/MeloTTS/blob/main/docs/training.md)

Excuse me again, I would like to ask how I would collect this dataset if I only want to train one person's tone, and if there is anything I need to pay attention to
